### PR TITLE
Borrowcommand tp

### PR DESCRIPTION
**Pull request by tp757**
Added the borrow command/ fixed bug on offer command/ added showmyloans command - added as this will distinguish loan from normal account type

### CODE REVIEW

File: newbank/server/Commands/BorrowCommand.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            .map(customer::getAccountFromNumber)
+            .filter(account -> !account.getAccountType().equals(Account.AccountType.LENDING))
+            .collect(Collectors.toList());
+    return accounts.size() == 0
+        ? null
+        : accounts.size() == 1 ? accounts.get(0) : queryAccount(customer, response, accounts);
+  }
+
+  private static Account queryAccount(
+      Customer customer, NewBankCommandResponse response, List accountList) {
```
Comment (inariDeeTee) [inline newbank/server/Commands/BorrowCommand.java]: List should take the parameter Account here.

Comment (tp757) [reply newbank/server/Commands/BorrowCommand.java]: Hi, just changed, also I made sure the crediting account was GBP

**Comment by inariDeeTee** [comment]
Hi, looks good so far. The major issue I noticed is that customers are able to borrow from their own offers which doesn't really make sense to allow. Could you add a check to prevent this - something like !borrowingCustomer.equals(offeringCustomer)?

**Comment by tp757** [comment]
I fixed that issue, was quite tricky as we did not store which customer makes which offer. Also there was not a way to use the equals method as they are different objects, in the end the only thing that are unique are the account numbers, so if both objects have the same, they must be the same person.

**Comment by hs2158** [comment]
Yes, I thought it was going to be fiddly because of the way Offer is structured but I was sure you could find a way! Should be ready to go now.

**Review by inariDeeTee — APPROVED** [review]

**Pull request merged**